controller/controller-board/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
the license looks to be required from a gulp project, let's try git submodule instead of copying all the code to here